src/lib/dapp/index.ts

```diff
@@ -107,54 +113,31 @@ export class DAppConnector {
         projectId: this.projectId,
         metadata: this.dAppMetadata,
       })
-      const existingSessions = await this.checkPersistedState()
-
-      if (existingSessions.length) {
-        await this.onSessionConnected(existingSessions.pop()!)
+      const existingSessions = this.walletConnectClient.session.getAll()
 
-        while (existingSessions.length) {
-          this.disconnect(existingSessions.pop()!.topic)
-        }
-      }
-
-      this.walletConnectClient.on('session_event', (event) => {
-        // Handle session events, such as "chainChanged", "accountsChanged", etc.
-        alert('There has been a session event!')
-        console.log(event)
-      })
-
-      this.walletConnectClient.on('session_update', ({ topic, params }) => {
-        // Handle session update
-        alert('There has been a update to the session!')
-        const { namespaces } = params
-        const _session = this.walletConnectClient!.session.get(topic)
-        // Overwrite the `namespaces` of the existing session with the incoming one.
-        const updatedSession = { ..._session, namespaces }
-        // Integrate the updated session state into your dapp state.
-        console.log(updatedSession)
-      })
+      if (existingSessions)
+        this.signers = existingSessions.flatMap((session) => this.createSigners(session))
 
       this.walletConnectClient.on('session_delete', (pairing) => {
-        console.log(pairing)
         this.signers = this.signers.filter((signer) => signer.topic !== pairing.topic)
         this.disconnect(pairing.topic)
-        // Session was deleted -> reset the dapp state, clean up from user session, etc.
-        console.log('Dapp: Session deleted by wallet!')
       })
 
       this.walletConnectClient.core.pairing.events.on('pairing_delete', (pairing) => {
-        // Session was deleted
-        console.log(pairing)
```

Comment:
I would like to see these console.logs gone too, but I don't think these console.logs should be removed in this PR. Maybe a new PR should be made to add a more production-ready logger instead of using console.log. Non-blocking comment though, feel free to resolve and not fix